tests/ice40/add_sub.ys

```diff
@@ -0,0 +1,7 @@
+synth_ice40
```

Comment:
Actually, can you replace the first `synth_ice40` call with `synth -coarse` which causes it to do technology independent synthesis. That way you are checking that technology-independent synthesis is equivalent to technology-specific, rather than checking `synth_ice40` against itself.

May I suggest this template for all tests:
```
hierarchy -top top
synth -run coarse # technology-independent coarse grained synthesis
equiv_opt -assert -map +/ice40/cells_sim.v synth_ice40 # equivalency check same as technology-dependent fine-grained synthesis
design -load postopt # load the post-opt design (otherwise equiv_opt loads the pre-opt design)
cd top # Constrain all select calls below inside the top module
select -assert-count 11 t:SB_LUT4
select -assert-count 6 t:SB_CARRY
select -assert-none t:SB_LUT4 t:SB_CARRY %% t:* %D # Check that there is no other cells except for these two. Explained, this select command means: select all SB_LUT4 cells as first element of stack. Then push all SB_CARRY cells as second element of stack. "%%" means to find union of all elements of stack. "t:*" selects all cells, and "%D" subtracts the top-1 element (i.e. all SB_LUT4 and SB_CARRY) from the top element (all cells) and ensures that no cells are leftover.
```

Reply:
done